src/component/Header.jsx

```diff
@@ -1,8 +1,58 @@
 import React from "react";
-
+import "../assets/myStyle/header.scss";
```

Comment:
Why do you importing header.scss here? Since you already import this file into index there is no need to load it again in the component file.

Reply:
Ok ,  I`ll fix it soon.